src/schematools/contrib/django/management/commands/change_dataset.py

```diff
@@ -1,21 +1,23 @@
+import argparse
 from argparse import ArgumentTypeError
 from distutils.util import strtobool
+from typing import Any
 
 from django.core.exceptions import ValidationError
 from django.core.management import BaseCommand, CommandError
 
 from schematools.contrib.django.models import Dataset
 
 
-def _strtobool(value):
+def _strtobool(value: Any) -> bool:
```

Comment:
Judging by its name `strtobool` this function converts a `str` to a `bool`. The docstring of the function you delegate to confirm that: _Convert a string representation of truth to true (1) or false (0)._. Hence the type annotation of `value` should not be `Any`, but `str`

Reply:
Fixed in 94e532695932246f7fa2a57e090f18b01043a59d.